suites/nautilus/rgw/sanity_containerized_rgw.yaml

```diff
@@ -572,56 +469,6 @@ tests:
 
    # Encryption tests
 
-   - test:
```

Comment:
if you remove the purge and install here, following all tests will have encryption enabled. this completely changes the scenario.